imu_sensor_notes.md

```diff
@@ -94,13 +94,13 @@ The `cal_acc_horizontal_offset` is the offest (difference) that the Joy-Con/Pro-
 
 The `cal_acc_horizontal_offset` is always the same. Advise [here](spi_flash_dump_notes.md#6-axis-and-stick-device-parameters) for the values each model (JC Left, JC Right, Pro).
 
-The `cal_acc_coeff` is used for the equations and it's always `x4000` (`16384`).
+The `cal_acc_coeff` is used for the equations.
```

Comment:
The `cal_acc_coeff` is used for calibrating sensitivity. The default uncalibrated value for ±8G sensitivity is `x4000` (`16384`).